pkg/asset/agent/image/baseiso.go

```diff
@@ -129,6 +130,10 @@ func (i *BaseIso) Generate(dependencies asset.Parents) error {
 			return nil
 		}
 		if !errors.Is(err, &exec.Error{}) { // Already warned about missing oc binary
+			if strings.Contains(err.Error(), "no such file or directory") {
+				// if image extract failed to extract the iso that architecture may be missing from release image
+				return errors.Wrapf(err, "failed to extract a base ISO image matching %s", archName)
```

Comment:
We can be more explicit about the problem here I think. e.g. "Base ISO for %s not found in release image. Check release image architecture."

Reply:
Good point, updated.